src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -538,6 +538,18 @@ internal void ApplyConflictResolutionEdits(IInlineRenameReplacementInfo conflict
                 }
             }
 
+            private string GetWithoutAttributeSuffix(string text, Document document)
```

Comment:
text seems not used and you can always get it from _session. so, it should either made static and not use _session or don't need to pass in.

Reply:
if it is going to be made static, then might be better to accept isCaseSensitive rather than document since that's only reason document is passed in.

also, if not going to make static, might be better to make it just local function rather than regular function since i dont think it will be used any other place except here.